### DISCUSSION TITLE
部署了项目发现下载的代码和Demo的不一样

### DISCUSSION THREAD

**Discussion by MikeJiangInno** [jd-opensource/drip-form#276] [Q&A]
不知道是不是我下载的版本有问题？如果是请告诉我。
我看到Demo中有配置联动。但是我运行的项目中。
没有配置联动。
![image](https://github.com/jd-opensource/drip-form/assets/137243733/c8ba1fe4-123f-4337-8465-9e9dd2629ac7)
Demo站点选择器中可以设置API获取数据，但是下载的代码版本没有这个功能。
![image](https://github.com/jd-opensource/drip-form/assets/137243733/3f52fda3-68c1-4d4d-aca7-6a040a9a81c1)
Demo站点有自定义组件，但是我下载的版本没有
![image](https://github.com/jd-opensource/drip-form/assets/137243733/3846c776-93ab-4260-8fd8-f460125155f0)
是我下载的版本有问题吗？还是需要在哪里配置？

**Comment by mengshang918** [comment]
试一下最新的alpha版本。

**Reply by MikeJiangInno** [reply]
alpha 版本，请问在哪可以下载？

**Reply by mengshang918** [reply]
npm包 @jdfed/form-generator 0.9.0-alpha.20试一试

**Reply by MikeJiangInno** [reply]
谢谢你。问题解决了。
